src/query/scan_manager.h

```diff
@@ -302,6 +302,13 @@ struct scan_pos
   QFILE_TUPLE_POSITION ls_tplpos;	/* List file index scan position */
 };				/* Scan position structure */
 
+typedef struct scan_agl SCAN_AGL;
```

Comment:
[Q] Does AGL stands for Aggregation List?

Reply:
agl stands for Aggregation Lookup.